<ui version="4.0" >
 <author>José Manuel Díez Botella</author>
 <class>tiposiva</class>
 <widget class="QDialog" name="tiposiva" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edición / Alta tipos de IVA</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Código:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="CodigoLineEdit" >
         <property name="maximumSize" >
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength" >
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Tipo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tipolineEdit" >
         <property name="maximumSize" >
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength" >
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>R.E.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="relineEdit" >
         <property name="maximumSize" >
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength" >
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Descripción</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DescripcionLineEdit" >
         <property name="maxLength" >
          <number>80</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="BotonGuardar" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Guardar</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonEliminar" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Eliminar</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonCancelar" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Cancelar</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BotonSalir" >
       <property name="text" >
        <string>&amp;Salir</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tabla" />
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>BotonGuardar</tabstop>
  <tabstop>BotonEliminar</tabstop>
  <tabstop>BotonCancelar</tabstop>
  <tabstop>BotonSalir</tabstop>
  <tabstop>CodigoLineEdit</tabstop>
  <tabstop>tipolineEdit</tabstop>
  <tabstop>relineEdit</tabstop>
  <tabstop>DescripcionLineEdit</tabstop>
  <tabstop>tabla</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>BotonSalir</sender>
   <signal>clicked()</signal>
   <receiver>tiposiva</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>372</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel" >
     <x>420</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
